python/cuda_parallel/cuda/parallel/experimental/iterators/__init__.py

```diff
@@ -19,15 +35,74 @@ def CacheModifiedInputIterator(device_array, modifier):
 
 
 def ConstantIterator(value):
-    """Returns an Iterator representing a sequence of constant values."""
+    """Returns an Iterator representing a sequence of constant values.
+
+    Similar to https://nvidia.github.io/cccl/thrust/api/classthrust_1_1constant__iterator.html
+
+    Example:
+        The code snippet below demonstrates the usage of a ``ConstantIterator``
+        representing the sequence ``[10, 10, 10]``:
```

Comment:
It might be better to move this to the example code itself, so that there's less risk of the two going out of sync, i.e., https://github.com/NVIDIA/cccl/pull/3311/files#r1909351881

(ditto in other places)

What do you think?

Reply:
Eh, after looking at the rendered docs, I convinced myself this is  good as is. Ideally we'd write a doctest here to keep the code closer to the commentary, but that can be future work.

Sorry for the confusion!